docs/source/index.rst

```diff
@@ -29,7 +29,7 @@
 
    api/hessian.rst
    api/fisher.rst
-   api/random_projection.rst
+   api/projection.rst
```

Comment:
I think you may also add
```rst
.. autofunction:: dattri.func.random_projection.arnoldi_project
```
to `projection.rst`

Reply:
fixed, thanks!